### PR TITLE
Add plot demonstrating effect of transformation options

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
The "none" transformation values are not necessarily what we'll have in our system, as this is a random gene (processed with SCAN) and no quantile normalization was performed.

### CODE REVIEW

File: docs/main_text.md [hunk 1]
```diff
@@ -225,8 +225,11 @@ It assumes that the data are normally distributed.
 * **Zero to one:** Rows are scaled to values `[0,1]` using the [`MinMaxScaler`](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html) from [`scikit-learn`](http://scikit-learn.org/stable/index.html).
 We expect this transformation to be most useful for certain machine learning applications (e.g., those using cross-entropy as a loss function).
 
-TODO: screenshots? plots that show effect of different scaling options?
+In the plot below, we demonstrate the effect of different scaling options on gene expression values (using a random selected human dataset, microarray platform, and gene):
```
Comment (cgreene) [inline docs/main_text.md]: Randomly selected?

**Comment by jaclyn-taroni** [comment]
Okay @cgreene - resized the image and also dropped the legend
